src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/LagPortSpec.groovy

```diff
@@ -516,6 +516,70 @@ class LagPortSpec extends HealthCheckSpecification {
         lagPort && northboundV2.deleteLagLogicalPort(sw.dpId, lagPort)
     }
 
+    @Tidy
+    def "Able to create/update LAG port with duplicated port numbers on the #sw.hwSwString switch"() {
+        when: "Create LAG port with duplicated port numbers"
+        def switchPortToCreate = topology.getAllowedPortsForSwitch(sw)[0]
+        def swAddress = northbound.getSwitch(sw.dpId).address
+        def portListToCreate = [switchPortToCreate, switchPortToCreate]
+        def createPayload = new LagPortRequest(portNumbers: portListToCreate)
+        def lagPortCreateResponse = northboundV2.createLagLogicalPort(sw.dpId, createPayload)
+
+        then: "Response shows that LAG port created successfully"
+        with(lagPortCreateResponse) {
+            logicalPortNumber > 0
+            portNumbers == [switchPortToCreate]
+        }
+        def lagPort = lagPortCreateResponse.logicalPortNumber
+
+        and: "Northbound shows that LAG port created"
+        with(northboundV2.getLagLogicalPort(sw.dpId)[0]) {
+            logicalPortNumber == lagPort
+            portNumbers == [switchPortToCreate]
+        }
+
+        and: "GRPC shows that created port exist in a list of all LAG ports"
+        with(grpc.getSwitchLogicalPortConfig(swAddress, lagPort)) {
+            logicalPortNumber == lagPort
+            name == "novi_lport" + lagPort.toString()
+            portNumbers == [switchPortToCreate]
+            type == LogicalPortType.LAG
+        }
+
+        when: "Update the LAG port with duplicated port numbers"
+        def switchPortToUpdate = topology.getAllowedPortsForSwitch(sw)[1]
+        assert switchPortToCreate != switchPortToUpdate
+        def portListToUpdate = [switchPortToUpdate, switchPortToUpdate]
+        def updatePayload = new LagPortRequest(portNumbers: portListToUpdate)
+        def lagPortUpdateResponse = northboundV2.updateLagLogicalPort(sw.dpId, lagPort, updatePayload)
+
+        then: "Response shows that LAG port updated successfully"
+        with(lagPortUpdateResponse) {
+            logicalPortNumber == lagPort
+            portNumbers == [switchPortToUpdate]
+        }
+
+        and: "Northbound shows that LAG port updated successfully"
+        with(northboundV2.getLagLogicalPort(sw.dpId)[0]) {
+            logicalPortNumber == lagPort
+            portNumbers == [switchPortToUpdate]
+        }
+
+        and: "GRPC shows that LAG port updated successfully"
+        with(grpc.getSwitchLogicalPortConfig(swAddress, lagPort)) {
+            logicalPortNumber == lagPort
+            name == "novi_lport" + lagPort.toString()
+            portNumbers == [switchPortToUpdate]
+            type == LogicalPortType.LAG
+        }
+
+        cleanup:
+        lagPort && northboundV2.deleteLagLogicalPort(sw.dpId, lagPort)
+
+        where:
+        sw << getTopology().getActiveSwitches().unique(false)
```

Comment:
There is no sense to iterate over all switches. We just need to test that code validation works fine, it doesn't depend on a model of a switch.